ecosystem/platform/server/app/jobs/kyc_complete_job.rb

```diff
@@ -6,9 +6,13 @@
 class KYCCompleteJobError < StandardError; end
 
 class KYCCompleteJob < ApplicationJob
-  # Ex args: { user_id: 32, inquiry_id=inq_syMMVRdEz7fswAa2hi }
+  # Ex args: { user_id: 32, inquiry_id=inq_syMMVRdEz7fswAa2hi, external_id: 141bc487-e025-418e-6e32-b7897060841c }
   def perform(args)
-    user = User.find(args[:user_id])
+    user = if args[:user_id].present?
+             User.find(args[:user_id])
+           else
+             User.where(external_id: args[:external_id]).first!
```

Comment:
Just to confirm, the intention is 500 if user_id is nil and no user exists with corresponding external_id?

Reply:
yeah; this could only ever happen if they're manually changing things